Rdmp.Dicom/TagPromotionSchema/TagColumnAdder.cs

```diff
@@ -82,7 +82,7 @@ public void Check(ICheckNotifier notifier)
             var db = GetDatabase();
             try
             {
-                var cSharpType = db.Server.GetQuerySyntaxHelper().TypeTranslater.GetCSharpTypeForSQLDBType(_datatype);
+                db.Server.GetQuerySyntaxHelper().TypeTranslater.GetCSharpTypeForSQLDBType(_datatype);
                 notifier.OnCheckPerformed(new CheckEventArgs("Datatype is compatible with TypeTranslater",CheckResult.Success));
```

Comment:
Add the cSharpType into the check message e.g. `$"Datatype {_datatype} is compatible with TypeTranslater as csharp type {cSharpType}"`  (assuming that var is a Type otherwise if it is a `Guess` then pass the actual Guess.Type to that message?